README.md

```diff
@@ -1,9 +1,9 @@
 # Crowbar: Client
 
-This is an experimental implementation to replace the current CLI scripts of
-the crowbar barclamps. This tool is intended to be installable standalone,
-that way everybody who got access to the Crowbar admin network should be able
-to manage Crowbar from remote.
+A complete reimplementation of the old CLI scripts of the Crowbar Barclamps.
```

Comment:
>  Isn't there a "This is.." missing?

I thought the "This is" is obvious, so I removed it. But if you prefer that, I will put it back

> This tool is a complete reimplementation of the old CLI scripts of Crowbar. It is intended...

even though the next sentence starts with "This tool..." as well?